api/bot/slackbot/v1/slackbot.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package clutch.bot.slackbot.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/bot/slackbot/v1;slackbotv1";
+
+import "api/v1/annotations.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+service SlackBotAPI {
+  rpc Event(EventRequest) returns (EventResponse) {
+    option (google.api.http) = {
+      post : "/v1/bot/slackbot/event"
+      body : "*"
+    };
+    option (clutch.api.v1.action).type = CREATE;
+  }
+}
+
+// Information on the bot user
+message Bot {
+  // bot id
+  string id = 1;
+  // if the bot is deleted
+  bool deleted = 2;
+  // bot name
+  string name = 3;
+  // timestamp of when the bot app was last updated
+  int64 updated = 4;
```

Comment:
```suggestion
  int64 last_updated = 4;
```

Reply:
`updated` is the field name that the Slack API sends to our endpoint so can't rename
example
```
{
        "id": "B061F7JD2",
        "deleted": false,
        "name": "beforebot",
        "updated": 1449272004,
        "app_id": "A161CLERW",
        "user_id": "U012ABCDEF",
        "icons": {
            "image_36": "https://...",
            "image_48": "https://...",
            "image_72": "https://..."
        }
    }
```

Reply:
:(